extensions/investspec/investspec.py

```diff
@@ -25,6 +26,23 @@ def format_type_string(arg_type):
     Returns:
         formatted string that links to a description of the input type(s)
     """
+    # some types need a more user-friendly name
+    # all types are listed here so that they can be marked up for translation
+    type_names = {
```

Comment:
I have a stupid question, are these types defined somewhere on the `natcap.invest` side? I looked around but didn't find a similarly valid types structure... I was mostly curious because I was wondering if we could take advantage of a single source of truth on the `natcap.invest` side in case types were added in the future. Not that'd we really be messing with these types all that much. Sorry about this ramble.

Reply:
I don't believe there is... it would make sense to import them from `natcap.invest`. `natcap.invest` itself has no use for _translations_ of the type names, but since they're used here and would be used in the workbench, we could centralize them there.

Reply:
Okay, it's fine to leave as is for this PR then. I think it's fine to wait until this actually pops up as an issue before centralizing things in `natcap.invest`.